### ISSUE TITLE
Training ssd_resnet_50_fpn_coco from scratch

### ISSUE THREAD

**Issue by mmanoharahere** [tensorflow/models#7948]
Hello,

I'm trying to train ssd_resnet_50_fpn_coco from scratch on COCO itself. I'm aware that the checkpoints are available, but this is for an experiment. The loss is not converging and the accuracy is very far from the reported accuracy.
Are there any tips to train SSD resnet from scratch?
Thank you!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
